html/includes/table/ports.inc.php

```diff
@@ -38,29 +38,28 @@
 
 $sql .= ' LEFT JOIN `devices` AS `D` ON `ports`.`device_id` = `D`.`device_id`';
 
-if (!empty($_POST['hostname'])) {
-    $where .= ' AND `D`.`hostname` LIKE ?';
-    $param[] = '%' . $_POST['hostname'] . '%';
+if (!empty($vars['hostname'])) {
+    $where .= ' AND (D.hostname LIKE "%i'.$vars['hostname'].'%" OR D.sysName LIKE "%'.$vars['hostname'].'%")';
```

Comment:
You need to put this back to `$where` and `$param`

Reply:
Ok I'll fix that;
Do you want the same for devices then? 
Because it's (almost) the same query.

Reply:
Yeah I saw that, you can fix if you like but it's not necessary. Just don't regress the code :)